.travis.yml

```diff
@@ -92,16 +92,12 @@ install:
       # This block only runs if the cache isn't present.
       if [ ! "$(ls -A $DEPSPREFIX)" ]; then
         mkdir -p work && pushd work
-        wget 'https://github.com/zeromq/zeromq4-1/releases/download/v4.1.5/zeromq-4.1.5.tar.gz' -O - |tar -zxf -
-        wget 'https://github.com/zeromq/czmq/releases/download/v3.0.2/czmq-3.0.2.tar.gz' -O - |tar -zxf -
         wget 'https://github.com/fontforge/libspiro/releases/download/0.5.20150702/libspiro-0.5.20150702.tar.gz' -O - | tar -zxf -
         wget 'https://github.com/fontforge/libuninameslist/releases/download/20190305/libuninameslist-dist-20190305.tar.gz' -O - | tar -zxf -
         git clone --branch v1.0.2 https://github.com/google/woff2 --depth 1
         wget --tries 1 "http://download.savannah.gnu.org/releases/freetype/freetype-$FTVER.tar.gz" || \
           wget "https://sourceforge.net/projects/freetype/files/freetype2/$SFFTVER/freetype-$FTVER.tar.gz"
 
-        pushd zeromq-4.1.5 && ./configure --prefix=$DEPSPREFIX && make && make install && popd
-        pushd czmq-3.0.2 && CFLAGS="$CFLAGS -Wno-format-truncation" ./configure --prefix=$DEPSPREFIX && make && make install && popd
         pushd libspiro-0.5.20150702 && ./configure --prefix=$DEPSPREFIX && make && make install && popd
```

Comment:
Not great to be sneaking in cleanup from one mass-removal into another, but acceptable I suppose.